qiskit_metal/_gui/widgets/create_component_window/parameter_entry_window.py

```diff
@@ -442,8 +443,10 @@ class definition
                 return memtup[1]
 
 
-def create_default_from_type(my_t: type):
+def create_default_from_type(my_t: type, param_name=None):
     """Create default values for a given type"""
+    if param_name is not None:
+        return param_name + "-" + str(random.randint(0, 1000))
```

Comment:
Below these lines in the list of `if` statements, I still see two instances of `fake-param` being returned.

Reply:
Hi Jeremy, 
Those should stay as fake-param. 
Zlatko just wanted the "name" argument default changed.

Reply:
you mean this is for when we add a new row to a dictionary? Yes, sounds good